script/test-setup

```diff
@@ -5,13 +5,34 @@
 const fs = require('fs')
 const path = require('path')
 const cp = require('child_process')
-const { getUserDataPath } = require('./dist-info')
+const { getUserDataPath, getDistPath, getProductName } = require('./dist-info')
+
+const isFork = process.env.TRAVIS_SECURE_ENV_VARS !== 'true'
+
+if (process.platform === 'darwin' && process.env.TRAVIS && !isFork) {
+  const archive = `${getDistPath()}/${getProductName()}.app`
+  try {
+    console.log('validating signature of Desktop app')
+    cp.execSync(`codesign -dv --verbose=4 '${archive}'`)
+  } catch (err) {
+    process.exit(1)
+  }
+  console.log('\n\n')
+}
+
+if (process.platform === 'darwin' && process.env.TRAVIS) {
```

Comment:
Making this macOS specific because Windows doesn't have these prompts for when apps access to the credential store, but might be worth making this consistent...